### ISSUE TITLE
Improved JAWS and NVDA support

### ISSUE THREAD

**Issue by amitnavale** [atlassian/react-beautiful-dnd#1288]
### Expected behavior
With Screen reader, movement of draggable item with arrow keys does not work.

### Actual behavior
Screen reader (Tested with JAWS and NVDA) arrow keys defaults to read next character. Because of this draggable items content get read instead of moving item to next position.

### Steps to reproduce
Open screen reader
navigate to example
press space bar for draggable item and press arrow keys
### Suggested solution?
in a drag mode (pressing of space bar on draggable item) capture arrow keys and prevent default behaviour

### What version of `React` are you using?
16.8.4

### What version of `react-beautiful-dnd` are you running?
11.0.0

### What browser are you using?
chrome 73.0.3683.103
IE11

### Demo
this reproducible on the stories we have on the examples link
https://react-beautiful-dnd.netlify.com/iframe.html?id=single-vertical-list--basic

This is an awesome library with keyboard support. Thanks for it. 
Extending it to be functional with screen readers will widen its impact exceptionally.

**Comment by alexreardon** [comment]
This is a shame for sure! This was tested about a year ago and we found it worked. This is a fiddly area to get correct. Would you be comfortable spiking a few things to see if they work?

**Comment by alexreardon** [comment]
Shipping in 12.0 #1317
